regexes/oss.yml

```diff
@@ -1160,6 +1160,6 @@
 ###########
 # Linux (Generic)
 ###########
-- regex: 'Linux(?:OS)?[^a-z]'
+- regex: 'Linux(?:OS)?[^a-z]|Cinnamon/(?:\d+[\.\d]+)'
```

Comment:
Do you know what that `Cinnamon` in an useragent stands for?

Reply:
current desktop environment (as an example, this is KDE Plasma)
https://en.wikipedia.org/wiki/Cinnamon_(desktop_environment)
https://github.com/linuxmint/Cinnamon

Reply:
thanks 👌